sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -27,6 +27,9 @@ if("cuda" IN_LIST SYCL_ENABLE_PLUGINS)
 endif()
 if("hip" IN_LIST SYCL_ENABLE_PLUGINS)
   set(UR_BUILD_ADAPTER_HIP ON)
+  set(UR_HIP_INCLUDE_DIR "" CACHE FILEPATH "Custom ROCm HIP include dir")
+  set(UR_HIP_HSA_INCLUDE_DIR "" CACHE FILEPATH "Custom ROCm HSA include dir")
+  set(UR_HIP_LIB_DIR "" CACHE FILEPATH "Custom ROCm HIP library dir")
```

Comment:
The documentation change mentiones inferring these from `SYCL_BUILD_PI_HIP_ROCM_DIR`, is that actually implemented though?

Reply:
That is actually a good point. Somehow I got confused with the mirrors of UR variables:
* `SYCL_BUILD_PI_HIP_ROCM_DIR` <-> `UR_HIP_ROCM_DIR`
* `PI_HIP_INCLUDE_DIR` <-> `UR_HIP_INCLUDE_DIR`
* `PI_HIP_HSA_INCLUDE_DIR` <-> `UR_HIP_HSA_INCLUDE_DIR`
* `PI_HIP_LIB_DIR` <-> `UR_HIP_LIB_DIR`

I've changed it to mention `PI_` and pointed users to `UR_` docs.#
c4bae485aa3215510e84fc9b85b0659936e662b5

Reply:
Okay, so if I set `SYCL_BUILD_PI_HIP_ROCM_DIR` how is `UR_HIP_ROCM_DIR` getting set or does it not get set?

Reply:
Those variables are not linked together. UR mirrors them and will require stetting them separately. See this comment for context: https://github.com/oneapi-src/unified-runtime/pull/1170#discussion_r1420622828 (on a rebased PR, sorry).

Reply:
@kbenzie I've moved the variables to `plugins/hip` so they can have meaningful defaults. This should do the trick now.